doc/sailfish-webview-rawwebview.qdoc

```diff
@@ -10,8 +10,34 @@
   \ingroup sailfishos-webview
```

Comment:
Now that looking code and documentation again I am leaning towards combining RawWebView documentation to the WebView docs and leaving RawWebView as an \internal doc (if any). The RawWebView is more like an implementation detail.

This would imply that we should document WebView inherited from an Item and all adding RawWebView and QuickMozView properties, methods, and signals there. Other of scope of this PR for sure.

For this PR having RawWebView as \internal (or even gone) and WebView inhering from Item would be enough I think.

Reply:
I struggled to explain how RawWebVew was supposed to be used, so I think this is the right call. I've removed the page and moved its properties to the WebVew docs.